spec/use_cases/performance_platform/publish_locations_ips_spec.rb

```diff
@@ -1,13 +1,13 @@
 describe UseCases::PerformancePlatform::PublishLocationsIps do
-  subject do
+  subject(:use_case) do
     described_class.new(
       destination_gateway: s3_gateway,
       source_gateway: source_gateway
     )
   end
 
-  let(:source_gateway) { double(fetch_ips: s3_payload) }
-  let(:s3_gateway) { double }
+  let(:source_gateway) { instance_double(Gateways::Ips, fetch_ips: s3_payload) }
```

Comment:
Also never seen this one.  Don't see much benefit over using an anonymous double (but I don't know what the lint is)